doc/nvim-tree-lua.txt

```diff
@@ -200,7 +201,10 @@ Here is a list of the options available in the setup call:
     type: `boolean`
     default: false
 
-*nvim-tree.view*
+- |hide_root_folder|: hide the path of the current working directory on top of the tree
```

Comment:
badly resolved, add a `*nvim-tree.hide_root_folder*` above your option declaration and do not remove the `*nvim-tree.view*` before the view declaration :)